src/registrar/migrations/0095_remove_domaininformation_cisa_representative_email_and_more.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.10 on 2024-05-16 23:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("registrar", "0094_create_groups_v12"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="domaininformation",
+            name="cisa_representative_email",
+        ),
+        migrations.RemoveField(
+            model_name="domainrequest",
+            name="cisa_representative_email",
+        ),
+        migrations.AddField(
+            model_name="domaininformation",
```

Comment:
@CocoByte I feel a bit uncertain about storing it as a contact. We will make searching the contact table really difficult because we have all these repeat cisa representatives which will just keep growing and growing. Was the thought here mostly to keep it inline with things like AO? I

Reply:
It seemed reasonable to store it as a contact since we are doing that for things like "other contacts" and AO.  The issue of duplicates and orphaned contact entries is definitely an issue this approach brings up.  That being said, it seemed strange to duplicate contact fields within a Domain Request model when we have a Contact model readily available for this type of information.  Maybe this warrants further discussion.

Reply:
Hmmm....Ok, I think the safer thing for this then is to not store it as a contact and just use fields instead.  That's a more incremental approach.  If we want to make the architecture more fancy later it will be easy.  But trying to strip out usage of the Contact model later would be hard.  I'll update this.  It's an easy revision at this point.